### DISCUSSION TITLE
avtub.pro

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1519] [General]
Vidio tak dapat di putar

**Comment by Jokopentil** [comment]
Menggunakan adguard kiwi browser
Semua vidio tak dapat di putar

**Comment by realodix** [comment]
Adguard Android, Chrome, aman Bro

**Comment by Jokopentil** [comment]
![20230820_131511_rmscr.jpg](https://github.com/realodix/AdBlockID/assets/114223791/6850d8fa-b7f6-46cb-9a22-9a2a89ab07a4) tampilannya seperti ini
Apqkah mode siluman adguard berpengaruh

**Reply by Jokopentil** [reply]
https://avtub.pro/citra-tobrut-lepas-hijab-mandi-bugil-toge-id-69822726-mango-119742.html

**Reply by realodix** [reply]
Di Saya seperti ini, aman kok.. AG Android, Kiwi Browser 

> Apqkah mode siluman adguard berpengaruh

Harus diurutin sih apa yg bermasalah.. 

Coba matikan semua plugin aja dlu, biar kelihatan aslinya seperti apa. Atau jangan pakai mode siluman dlu, jadi kelihatan masalahnya dimana. 


<details>
  <summary>Screenshot</summary>
  
  ![Screenshot_2023-08-20-23-52-21-193_com kiwibrowser browser](https://github.com/realodix/AdBlockID/assets/1314456/d6c29144-793e-43da-81c3-29a999df1e2a)

</details>

**Comment by Jokopentil** [comment]
Di saya kok eror ya apakah setingan dns kiwi

Pada tanggal Min, 20 Agu 2023 23.59, Kei ***@***.***> menulis:

**Comment by Jokopentil** [comment]
Itu sudah fix saya coba di yandex dan saya nyalakan mode siluman
Ternyata itu penyebabnya

Pada tanggal Sen, 21 Agu 2023 00.01, Joko Pentil ***@***.***>
menulis:
